### PR TITLE
Answer open question 2

### PR DESCRIPTION
**Pull request by CBinh710**
Answer open question 2 "What HTML tags have property display: inline, block, inline-block?"

### CODE REVIEW

File: OpenQuestion/answer-question-2.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+In HTML and CSS, different tags have different default display properties:
```
Comment (hungdo-agilityio) [inline OpenQuestion/answer-question-2.md]: Please add the question as well

File: OpenQuestion/image.png
Comment (hungdo-agilityio) [inline OpenQuestion/image.png:1]: We can have multiple image, so the naming is not good

File: OpenQuestion/answer-question-2.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+   - Block-level elements always start on a new line and take up the full width available.
+   - Examples of block-level elements include `<div>`, `<p>`, `<h1>` to `<h6>`, `<ul>`, `<ol>`, `<li>`, `<table>`, and more.
+2. **Inline Elements**:
+   - Inline elements do not start on a new line and only take up as much width as necessary.
+   - Examples of inline elements include `<span>`, `<a>`, `<strong>`, `<em>`, `<img>`, `<input>`, and more.
+3. **Inline-Block Elements**:
+   - Inline-block elements behave like inline elements but allow you to set a width and height.
+   - They also respect top and bottom margins/paddings.
+   - Examples include `<button>`, `<label>`, and sometimes `<img>`.
+   ![alt text](image.png)
```
Comment (hungdo-agilityio) [inline OpenQuestion/answer-question-2.md]: `alt text` is not good, please recheck what is alt attribute

File: OpenQuestion/answer-question-2.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+In HTML and CSS, different tags have different default display properties:
+1. **Block-Level Elements**:
+   - Block-level elements always start on a new line and take up the full width available.
+   - Examples of block-level elements include `<div>`, `<p>`, `<h1>` to `<h6>`, `<ul>`, `<ol>`, `<li>`, `<table>`, and more.
+2. **Inline Elements**:
+   - Inline elements do not start on a new line and only take up as much width as necessary.
+   - Examples of inline elements include `<span>`, `<a>`, `<strong>`, `<em>`, `<img>`, `<input>`, and more.
+3. **Inline-Block Elements**:
+   - Inline-block elements behave like inline elements but allow you to set a width and height.
```
Comment (hungdo-agilityio) [inline OpenQuestion/answer-question-2.md]: Can inline elements and block element can set width and height?

File: OpenQuestion/answer-question-2.md (hunk 1 above)
Comment (CBinh710) [reply OpenQuestion/answer-question-2.md]: I added it

File: OpenQuestion/image.png
Comment (CBinh710) [reply OpenQuestion/image.png:1]: i renamed it

File: OpenQuestion/answer-question-2.md (hunk 2 above)
Comment (CBinh710) [reply OpenQuestion/answer-question-2.md]: I fixed it

File: OpenQuestion/answer-question-2.md (hunk 3 above)
Comment (CBinh710) [reply OpenQuestion/answer-question-2.md]: I answered it in new commit, thanks

**Pull request merged**